src/_guides/language/analysis-options.md

```diff
@@ -294,11 +294,8 @@ linter:
 ```
 
 {{site.alert.note}}
-  **If you disable any rules, then use key-value syntax to enable rules.**
-  For example, the preceding example can’t use `- await_only_futures`;
-  instead, it must use `await_only_futures: true`.
-  The reason it can’t use the dash shortcut (`-`) to enable `await_only_futures`
-  is that YAML doesn't support mixing list and key-value syntaxes.
+  Due to YAML restrictions, **you can't mix list and key-value syntax in the same rules entry.**
```

Comment:
Nit: ... same rules entry -> ... same `rules` entry 
(to match Kathy's proposal).

Reply:
Otherwise, LGTM